src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -28,6 +28,8 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
```

Comment:
please create separate PR in acq-models also with schema changes

Reply:
then don't forget to point acq-models to master after acq-model pr merged

Reply:
Sorry forgot that one, here's PR: https://github.com/folio-org/mod-orders/pull/948